packaging/gunicorn/ansible-runner-service.el7.spec

```diff
@@ -0,0 +1,105 @@
+%global srcname ansible-runner-service-dev
```

Comment:
EL7 is no longer supported platform for oVirt 4.4, so what do we need to have this spec for?

Reply:
I wanted to save the python2 spec file but will remove it.